software/deploy/deploy.go

```diff
@@ -170,6 +170,10 @@ func buildPushDockerImage(houstonClient houston.ClientInterface, c *config.Conte
 
 	imageHandler := imageHandlerInit(imageName)
 
+	if description != "" {
+		deployLabels = append(deployLabels, "io.astronomer.deploy.revision.description="+description)
```

Comment:
nit: silly question, how does this description pop up in the UI when it's a DAG-only deploy because CLI is only setting it on the image, and I am not sure we do an image deploy when the `--dags` flag is passed.

Reply:
Yeah, should we extend this flag to --dag-only command too? @Simpcyclassy

Reply:
Not a silly question at all. That is what this PR https://github.com/astronomer/astro-cli/pull/1716 is for. It would send the description to dag server which creates a revision on houston

Reply:
yes

Reply:
why are we setting io.astronomer.deploy.revision.description

Reply:
Because we are setting the description through the api request in the image